packages/ai-utils/tests/transforms.test.ts

```
@@ -1,5 +1,9 @@
 import { describe, it, expect } from 'vitest'
-import { transformNullsToUndefined } from '../src/transforms'
+import type { JsonSchemaNode } from '../src/transforms'
+import {
+  transformNullsToUndefined,
+  undoNullWidening,
+} from '../src/transforms'
```

ESLint

[error] `../src/transforms` import should occur before type import of `../src/transforms`

(import/order)